app/javascript/App/components/WatchlistItem/IcoData.js

```diff
@@ -2,7 +2,8 @@ import React from 'react'
 import dateFormat from 'dateformat'
 
 export default ({ coin }) => {
-  const { max_supply, symbol, ico_token_price_usd } = coin
+  const { max_supply, symbol, ico_token_price_usd} = coin
```

Comment:
You should use eslint and prettier if possible :)
https://github.com/coinfi/coinfi#coding-style-guide